src/vs/workbench/electron-browser/index.html

```diff
@@ -113,8 +113,20 @@
 			process.env['VSCODE_SHARED_IPC_HOOK'] = configuration.sharedIPCHandle;
 
 			// Get the nls configuration into the process.env as early as possible.
+			var nlsConfig = { availableLanguages: {} };
 			if (configuration.userEnv && configuration.userEnv['VSCODE_NLS_CONFIG']) {
-				process.env['VSCODE_NLS_CONFIG'] = configuration.userEnv['VSCODE_NLS_CONFIG'];
+				var config = configuration.userEnv['VSCODE_NLS_CONFIG'];
+				process.env['VSCODE_NLS_CONFIG'] = config;
+				try {
+					if (config) {
+						nlsConfig = JSON.parse(config);
+					}
+				} catch (e) {
+				}
+			}
+			var uiLanguage = nlsConfig.availableLanguages['*'];
+			if (uiLanguage) {
+				window.document.getElementsByTagName('html')[0].setAttribute('lang', uiLanguage)
```

Comment:
I am not sure if our language codes map directly to the expected language codes for the HTML attribute. The list of expected codes seem to be listed here: http://www.w3schools.com/tags/ref_language_codes.asp